_progress-bar.scss

```diff
@@ -0,0 +1,279 @@
+// progress-bar
+// ===
+
+// Configurable Variables
+// ---
+
+$progress-bar__progress-color: black !default;
+$progress-bar__success-color: $success-color !default;
+$progress-bar__error-color: $error-color !default;
+$progress-bar__background-color: $neutral-50 !default;
+
+
+// Dependencies
+// ---
+
+// Override Font Awesome's default path
+$fa-font-path: '/bower_components/font-awesome/fonts';
+
+@import 'stencil-variables/variables';
+@import 'level/level';
+@import 'font-awesome/scss/font-awesome';
+@import 'stencil-utils/dist/visibility/visibility';
+
+
+// Progress Bar root
+// ---
+
+.c-progress-bar {
+    display: flex;
+    align-items: center;
+    height: $tap-size;
+    margin: 0 15px;
```

Comment:
I'm not sure I like this margin. Might be a bit presumptuous to add spacers like this; I could foresee a design that wants  this component to be flush with the left most side of its container.

![screen shot 2015-08-07 at 2 42 10 pm](https://cloud.githubusercontent.com/assets/734535/9146549/855dfbe0-3d12-11e5-9f86-9f13d0c29825.png)